File: src/main/java/com/sudosoo/takeiteasy/dto/post/SetCategoryByPostRequestDto.java
package com.sudosoo.takeiteasy.dto.post;

import jakarta.validation.constraints.NotNull;
import lombok.AccessLevel;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class SetCategoryByPostRequestDto {
    @NotNull(message = "카테고리아이디를 입력해 주세요.")
    private Long categoryId ;
    @NotNull(message = "게시글아이디를 입력해 주세요.")
    private Long postId;
}

File: takeItEasy-application/src/main/kotlin/com/sudosoo/takeItEasy/dto/post/SetCategoryByPostRequestDto.kt
package com.sudosoo.takeItEasy.dto.post

import jakarta.validation.constraints.NotBlank

class SetCategoryByPostRequestDto(
    @NotBlank(message = "카테고리아이디를 입력해 주세요.")
    val categoryId: Long,
    @NotBlank(message = "게시글아이디를 입력해 주세요.")
    val postId: Long)
